eventmesh-security-plugin/eventmesh-security-auth-token/src/main/java/org/apache/eventmesh/auth/token/impl/auth/AuthTokenUtils.java

```diff
@@ -146,6 +81,50 @@ public static boolean authAccess(AclProperties aclProperties) {
         Set<String> groupTopics = TypeUtils.castSet(topics, String.class);
 
         return groupTopics.contains(topic);
+
+    }
```

Comment:
We generally won't leave a blank line at the end of a method.

Reply:
gotcha